frontend/src/utils/api.ts

```diff
@@ -64,7 +71,7 @@ window.encodeHash = function encodeHash() {
 };
 
 window.decodeHash = function decodeHash() {
-  const hashParams = {};
+  const hashParams: { [key:string]: string } = {};
```

Comment:
```suggestion
  const hashParams: { [key: string]: string } = {};
```

Reply:
Thanks for catching this, made the change